dev-docs/bidders/9dotsmedialtd.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: bidder
+title: 9dots media LTD
+description: Prebid 9dots media LTD Bidder Adaptor
+biddercode: 9dotsmedialtd
+pbjs: false
+pbs: true
+media_types: video, banner
+userIds: all
+fpd_supported: false
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+prebid_member: false
+ortb_blocking_supported: true
+multiformat_supported: will-bid-on-one
+floors_supported: false
+aliasCode: limelightDigital
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+
+| Name          | Scope    | Description                                                   | Example                | Type      |
+|:--------------|:---------|:--------------------------------------------------------------|:-----------------------|:----------|
+| `host`        | required | Ad network's RTB host                                         | `'<adserving domain>'` | `string`  |
```

Comment:
host cannot be required for PBS. The text below suggests that this only needed for PBJS, but that conflicts with the declaration above `pbjs:false`